web/src/lib/components/admin-page/settings/machine-learning-settings/machine-learning-settings.svelte

```diff
@@ -77,6 +77,40 @@
         </div>
       </SettingAccordion>
 
+      <SettingAccordion
+        key="duplicate-detection"
+        title="Duplicate Detection"
+        subtitle="Use CLIP embeddings to find likely duplicates"
+      >
+        <div class="ml-4 mt-4 flex flex-col gap-4">
+          <SettingSwitch
+            id="enable-duplicate-detection"
+            title="ENABLED"
+            subtitle="If disabled, exactly identical assets will still be de-duplicated."
+            bind:checked={config.machineLearning.duplicateDetection.enabled}
+            disabled={disabled || !config.machineLearning.enabled || !config.machineLearning.clip.enabled}
+          />
+
+          <hr />
+
+          <SettingInputField
+            inputType={SettingInputFieldType.NUMBER}
+            label="MAX DETECTION DISTANCE"
+            bind:value={config.machineLearning.duplicateDetection.maxDistance}
+            step="0.01"
+            min={0.001}
+            max={0.1}
+            desc="Maximum distance between two images to consider them duplicates, ranging from 0.001-0.1. Higher values will detect more duplicates, but may result in false positives."
+            disabled={disabled ||
+              !config.machineLearning.enabled ||
+              !config.machineLearning.clip.enabled ||
+              !config.machineLearning.duplicateDetection.enabled}
```

Comment:
might be able to just use a feature flag here instead. `$featureFlag.duplicateDetection`

Reply:
@NicholasFlamy What was the reason you didn't use the feature flag? I remember you have a PR put in for this, correct?

Reply:
I added the feature flag to the original backend PR. I didn't think to use the feature flag here (mainly I forgot about this part when I added that flag). So I think a feature flag could replace all of that. 
```suggestion
              $featureFlag.duplicateDetection
```
I haven't tested this, either I'll make the change on my test instance or if it looks good you could commit it and I'll test it, ***but*** I won't get to testing either option until tomorrow.